__tests__/app.test.js

```diff
@@ -231,6 +231,27 @@ describe("/api/articles", () => {
         });
     });
 	});
+  describe("FILTER QUERIES", () => {
+    test("returns 200 and articles filtered by topic", () => {
+      return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then(({body}) => {
+        body.articles.forEach((article) => {
```

Comment:
whenever asserting within a four each do a length check as an empty array will pass this test! .